acceptance/Cargo.toml

```diff
@@ -16,24 +16,24 @@ release = false
 [dependencies]
 delta_kernel = { path = "../kernel", features = [
   "default-engine",
-  "arrow_53",
+  "arrow_54",
```

Comment:
given how fraught arrow updates  have been, let's not do this in a PR that's also doing other stuff. 

Can we actually just make a different PR for the dependency version bumps (unless you need them for some functionality here)